test/k8sT/manifests/log-gatherer.yaml

```diff
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: log-gatherer
+  namespace: kube-system
+  labels:
+    k8s-app: cilium-test-logs
+spec:
+  selector:
+    matchLabels:
+      k8s-app: cilium-test-logs
+      kubernetes.io/cluster-service: "true"
+  template:
+    metadata:
+      annotations:
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+      labels:
+        k8s-app: cilium-test-logs
+        kubernetes.io/cluster-service: "true"
+    spec:
+      containers:
+      - args:
+        - "10000"
+        command:
+        - sleep
+        image: nebril/systemd:latest #TODO: move to cilium org
```

Comment:
I imagine the image is pretty stable now, so we could move it before we merge if you want.

Reply:
Don't have access rights to our dockerhub (security!). Will change this in followup PR

Reply:
neither do I! We'll do it later.